exporter/importables.go

```diff
@@ -752,7 +763,7 @@ var resourcesMap map[string]importable = map[string]importable{
 				displayName := d.Get("display_name").(string)
 				return applicationID == displayName
 			}
-			return pathString == "home" || pathString == "repos" || (pathString == "application_id" && !ic.Client.IsAzure())
```

Comment:
This is just a bug I guess? These fields don't seem to belong to databricks_service_principal?